### PR TITLE
Add tests for package name override

### PR DESCRIPTION
**Pull request by ramereth**
Signed-off-by: Lance Albertson <lance@osuosl.org>

# Description

[Describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves]

## Check List

- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -69,6 +83,12 @@ jobs:
         with:
           suite: ${{ matrix.suite }}
           os: ${{ matrix.os }}
+      - name: Print debug output on failure
+        if: failure()
+        run: |
+            set -x
+            sudo journalctl -l --since today
+            KITCHEN_LOCAL_YAML=kitchen.dokken.yml /usr/bin/kitchen exec ${{ matrix.suite }}-${{ matrix.os }} -c "journalctl -l"
```
Comment (Xorima) [inline .github/workflows/ci.yml:91]: if we keep doing this everywhere, why don't we extend the github action to do this by default?

File: resources/mod_ssl.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 include Apache2::Cookbook::Helpers
 unified_mode true
 
+property :mod_ssl_pkg, String,
+         default: 'mod_ssl',
+         description: ''
```
Comment (Xorima) [inline resources/mod_ssl.rb:6]: I think we should have some kind of description, even if it is just this:

```suggestion
         description: 'The name of the mod_ssl package'
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ramereth) [reply .github/workflows/ci.yml:91]: Should I keep this here for now and work on a separate PR which adds this to the action?

Comment (Xorima) [reply .github/workflows/ci.yml:91]: yeah, I think that is best.

**Review by Xorima — APPROVED** [review]

**Pull request merged**